data_steward/tools/delete_stale_test_datasets.py

```diff
@@ -0,0 +1,88 @@
+"""
+Delete stale datasets in test environment. 
+"""
+from datetime import datetime, timezone
+import logging
+import os
+
+# Project imports
+from utils import bq, pipeline_logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+def _check_project(bq_client):
+    """
+    Check if the project is set to test.
+
+    :param bq_client: Client
+    :return: None if the project is set properly
+    :raise: ValueError if project is not 'aou-res-curation-test'
+    """
+    test_project = 'aou-res-curation-test'
+
+    if bq_client.project != test_project:
```

Comment:
After today's request from security about identifiers in Github, I wonder if we should be scrubbing instances of full project names from the project.
```suggestion
    if 'test' not in bq_client.project:
```